Density/app/src/main/res/layout/facilities_activity.xml

```diff
@@ -99,6 +99,56 @@
                 android:layout_height="match_parent"
                 android:orientation="vertical">
 
+                <androidx.cardview.widget.CardView
+                    android:id="@+id/covidPolicyCard"
+                    style="@style/MyCardView"
+                    android:layout_width="fill_parent"
+                    android:layout_height="120dp"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginLeft="16dp"
```

Comment:
`marginStart` and `marginLeft` (and `marginEnd` == `marginRight`) are the same thing, so you don't need both—you should just use `marginStart` and `marginEnd`